modules/openapi-generator/src/main/resources/kotlin-client/enum_class.mustache

```diff
@@ -30,6 +31,7 @@ import kotlinx.serialization.*
  * Values: {{#allowableValues}}{{#enumVars}}{{&name}}{{^-last}},{{/-last}}{{/enumVars}}{{/allowableValues}}
  */
 {{#multiplatform}}@Serializable{{/multiplatform}}{{#kotlinx_serialization}}@Serializable{{#enumUnknownDefaultCase}}(with = {{classname}}Serializer::class){{/enumUnknownDefaultCase}}{{/kotlinx_serialization}}
+{{#moshi}}@JsonClass(generateAdapter = false){{/moshi}}
```

Comment:
To fix the empty space I think you need to format this line like this:
```
{{#moshi}}
@JsonClass(generateAdapter = false)
{{/moshi}}
```
I'm not 100% sure of this, but try it and see if it works.

Reply:
Thanks, it worked!